centaur/src/main/resources/standardTestCases/failing_fast_and_slow/fail.wdl

```diff
@@ -0,0 +1,78 @@
+version 1.0
+
+task shouldCompleteFast {
+    input {
+      Int a
+    }
+    command {
+        echo "The number was: ~{a}"
+    }
+    output {
+        Int echo = a
+    }
+    runtime {
+      docker: "ubuntu:latest"
+    }
+}
+
+task shouldCompleteSlow {
+    input {
+      Int a
+    }
+    command {
+        echo "The number was: ~{a}"
+        # More than 1 so this should finish second
+        sleep 2
+    }
+    output {
+        Int echo = a
+    }
+    runtime {
+      docker: "ubuntu:latest"
+    }
+}
+
+task failMeSlowly {
+    input {
+      Int a
+    }
+    command {
+        echo "The number was: ~{a}"
+        # Less than 2 so this should finish first
```

Comment:
This seems like an incredibly brave assumption, especially in PAPI-land

Reply:
Whilst Cromwell is incredibly brave you may have a point here.